itest/litd_custom_channels_test.go

```diff
@@ -2766,3 +2771,457 @@ func testCustomChannelsFee(_ context.Context,
 		"min_relay_fee: ", tooLowFeeRateAmount.FeePerKWeight())
 	require.ErrorContains(t.t, err, errFeeRateTooLow)
 }
+
+func assertNumHtlcs(t *testing.T, node *HarnessNode, expected int) {
```

Comment:
nit: could move these helpers to the `assets_test.go` file to de-clutter the main itest file a bit.